tools/packaging/kata-deploy/local-build/Makefile

```diff
@@ -25,11 +25,12 @@ all: cloud-hypervisor-tarball \
 	firecracker-tarball \
 	kernel-tarball \
 	qemu-tarball \
-	rootfs-image-tarball \
-	rootfs-initrd-tarball \
+	rootfs-tarballs \
 	shim-v2-tarball \
 	virtiofsd-tarball
 
+rootfs-tarballs: | rootfs-image-tarball rootfs-initrd-tarball
```

Comment:
@ryansavino 
As @fidencio mentioned before, I also think this still runs the jobs in parallel, so we need to split `all-parallel` into parallel job and serial job that is executed without `-j`.